.eslintrc.yaml

```diff
@@ -11,10 +11,12 @@ globals:
   BASE_URL: true
 
 ignorePatterns:
-  - auspice-client/
-  - docs/
-  - node_modules/
-  - static-site/
+  - "/*"
+  - "!/resourceIndexer"
+  - "!/scripts"
+  - "!/server.js"
+  - "!/src"
+  - "!/test"
```

Comment:
Do we want to keep linting `jest.config.js`?

Reply:
I don't see why not. Added back in force-push:

https://github.com/nextstrain/nextstrain.org/blob/6e1ec7f8d094214e3b3de66e8359f7d40f9d4282/.eslintrc.yaml#L15